var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -105,6 +109,8 @@ class Qt(Package):
     # depends_on("ogg", when='+multimedia')
 
     use_xcode = True
+    # parallel build fails on Sierra
+    parallel = False
```

Comment:
ok, here's a bug report https://bugreports.qt.io/browse/QTBUG-58038 but i can't figure out how to get to the commit `4dcfd90e4fd7d4c49138038dbbcbda8794a9fbff` which fixes it.